.github/workflows/oav-runner-tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: OAV Runner - Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - package-lock.json
+      - package.json
+      - tsconfig.json
+      - .github/shared
+      - .github/workflows/_reusable-eng-tools-test.yaml
+      - .github/workflows/lintdiff-test.yaml
+      - eng/tools/package.json
+      - eng/tools/tsconfig.json
+      - eng/tools/oav-runner/**
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash
+    working-directory: eng/tools/oav-runner
+
```

Comment:
```suggestion
```

You probably copied this from lintdiff, but I think it's dead code.  It's not needed in other test workflows.